src/server/js/services/firebase.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016 Google Inc.
```

Comment:
If you're adding a new directory, you'll need to append it to the watch list for `watch:node:scripts`.

Reply:
I think watch:node:scripts watches all of the "src/server/js"? Or does it work differently than I'm thinking?

Reply:
Yes, you're right, this lives inside `src/server/js`. All good, no changes needed.